libs/datasets/combined_datasets.py

```diff
@@ -216,15 +221,16 @@ def datasource_regions(
     ]
 )
 CDCNewVaccinesCompletedBoosterCountiesWithoutExceptions = datasource_regions(
-    CDCNewVaccinesCountiesDataset, exclude=[CDC_STATE_EXCLUSIONS, *CDC_COUNTY_EXCLUSIONS]
+    CDCNewVaccinesCountiesDataset,
+    exclude=[CDC_STATE_EXCLUSIONS, *CDC_COUNTY_EXCLUSIONS, NC_COUNTIES],
```

Comment:
Is there a reason not to just add NC_COUNTIES directly to CDC_COUNTY_EXCLUSIONS?

Reply:
right, that would make much more sense

done in 49aa642bbf27ef04df1df3e75f4903193cea8f70